lua/starfall/libs_sh/bit.lua

```diff
@@ -634,13 +634,27 @@ function bit_library.compress(s)
 	return ret
 end
 
---- Decompresses a string
+--- Decompresses a string using LZMA.
 -- @param string s Compressed string to decode
 -- @return string Decompressed string or nil if the input was invalid
-function bit_library.decompress(s)
+function bit_library.decompress(s, maxSize)
```

Comment:
I don't see any reason the user would need this. Keep it internal?

Reply:
I don't see any reason the user *shouldn't* have it. Might be useful if a chip was accepting LZMA compressed data from untrusted sources and wanted an even stricter memory and CPU time limit, for instance if it was decompressing a very large number of things in a single tick.

Reply:
Ok, be sure to add it to the doc comments above then